.travis.yml

```diff
@@ -151,9 +151,9 @@ stages:
   - test
   - test-java11
   - name: deploy
-    if: repo = lagom/lagom-samples # only deploy branches that are pushed directly to the repo
+    if: env(CP2_PLAY_PASSWORD) IS present # only run the "deploy" stage if the password is present
```

Comment:
@ignasi35 for some reason the `deploy` stage was executing for pull requests. This is now consistent with what we have in 1.5.x branch.

Reply:
nice! Thanks!